zips/zip-0246.rst

```diff
@@ -283,8 +284,8 @@ In the case that the transaction has no OrchardZSA Action Groups, ``orchard_dige
 T.4a: orchard_action_groups_digest
 ''''''''''''''''''''''''''''''''''
 
-A BLAKE2b-256 hash of the subset of OrchardZSA Action Groups information for all OrchardZSA Action Groups belonging to the transaction.
-For each Action Group, the following elements are included in the hash::
+A BLAKE2b-256 hash of a subset of OrchardZSA Action Group information for an OrchardZSA Action Group.
+The following elements of the Action Group are included in the hash::
```

Comment:
This is not a clarifying change, this completely changes the definition of the orchard_action_groups_digest. I believe that the previous definition was correct. See how `orchard_actions_compact_digest` is defined. I think that we should close this, and the implementation should be verified to be correct with respect to the prior wording.